tests/test_acceptance.py

```diff
@@ -58,6 +61,8 @@ def test_slurm(tmp_path: Path, scenario: Dict):
         assert td.is_dir(), "Invalid test directory"
         assert "Tests." in td.name, "Invalid test directory name"
 
+    assert log_file_path.exists(), f"Log file {log_file_path} does not exist"
```

Comment:
Let reformulated the message from `... does not exit` to `... was not created` to make it easy to understand what went wrong.